app/main.py

```diff
@@ -1,8 +1,50 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: float,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list[Car]) -> float:
+        income = 0.0
+        for car in cars:
+            if self.clean_power > car.clean_mark:
+                income += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+
+        return round(income, 1)  # pytest
+
+    def calculate_washing_price(self, car: Car) -> float:
+        clean_car = self.clean_power - car.clean_mark
+        service_level = clean_car * car.comfort_class  # service_level car
+        quality_service = service_level * self.average_rating
+        price = quality_service / self.distance_from_city_center
+        return price
+
+    def wash_single_car(self, car: Car) -> None:
+        if self.clean_power >= car.clean_mark:
```

Comment:
it will be true if  `self.clean_power == car.clean_mark`
I don't think this is how it should work

Reply:
+

Reply:
It's working like before, because without if `self.clean_power == car.clean_mark`  instead of `self.clean_power >= car.clean_mark` the program has 3 failed tests, you can check my code in your IDE.
About removing unnecessary comments - OK, I'll remove it.

Reply:
i mean its u have to use > or <, not >= or <=. Because in situation when `self.clean_power == car.clean_mark` ur program will work wrong.